advocacy_docs/partner_docs/ImpervaDataSecurityFabric/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+
+The Imperva Data Security Fabric (DSF), previously Imperva SecureSphere, system protects data with a hybrid security solution that is usable by all data types.  Imperva Data Security Fabric can help organizations reduce security risks from inside and outside sources, whether it be a careless mistake or malicious intent.
+
+The Imperva Data Security Fabric Agent is not demanding on the database server and has minimum overhead, which will allow your database server to continue to run without issue. It does provide constant, real-time monitoring of databases and database traffic.
+
+The diagram below shows how you can integrate the Imperva SecureSphere Agent with your EDB Postgres Advanced Server and PostgreSQL deployments.
```

Comment:
I know we have the diagram below, but a couple sentences just explaining what is going on in the image would be helpful as well.

Reply:
Added this- "The Imperva Data Security Fabric agent resides on your databases and monitors and continuously analyzes all data access activity on both database user accounts and privileged user accounts, in order to check for compliance or a violation of a security policy. "

Since I didn't test I am working on explaining the pieces as best as I can since I didn't use them.